pyaedt/edb.py

```diff
@@ -2529,20 +2528,11 @@ def variable_exists(self, variable_name):
             It returns a booleand to check if the variable exists and the variable
```

Comment:
```suggestion
            A Boolean indicating if the variable exists and the variable
```
What is a "variable" server? Should we drop the variable portion and conclude with "and the server that stores the variable."